sys-tuner/src/main.rs

```diff
@@ -0,0 +1,62 @@
+use std::fs;
+use unix_socket::UnixListener;
+
+#[cfg(any(not(unix), target_os = "macos"))]
+pub const SOLANA_SYS_TUNER_PATH: &str = "/tmp/solana-sys-tuner";
+
+// Use abstract sockets for Unix builds
+#[cfg(all(unix, target_os = "linux"))]
+pub const SOLANA_SYS_TUNER_PATH: &str = "\0/tmp/solana-sys-tuner";
+
+#[allow(dead_code)]
+#[cfg(target_os = "linux")]
+fn tune_system() {
+    use std::process::Command;
+    use std::str::from_utf8;
+    use thread_priority::*;
+
+    let output = Command::new("ps")
```

Comment:
It sorta sucks to rely on `ps` here.   How about just enumerate all directories in `/proc`, then read/parse `/proc/N/status`.  The first line in this file is "Name: solana-poh-serv`

Reply:
I'll take a look at it.